### ISSUE TITLE
Numerical instability in tutorial 'Fit a mesh via rendering' when optimizing using 'RGB image loss'

### ISSUE THREAD

**Issue by jwb95** [facebookresearch/pytorch3d#1130]
The code provided in the tutorial [Fit a mesh via rendering](https://pytorch3d.org/tutorials/fit_textured_mesh) has a bug.
That is in the last section 'Mesh and texture prediction via textured rendering' mesh-vertices and respective texture are optimized via 5 losses "silhouette", "edge", "normal", "laplacian" plus an additional "rgb"-loss. In the previous section optimizing via the first 4 losses works just fine, but the "rgb"-loss yields nan after a few optimization steps. Attenuating the learning rate did not help.

To reproduce the error simply open the colab notebook provided on the webpage and run all cells.

**Comment by jwb95** [comment]
Running the code today worked. I have no idea what has caused the issue.

**Comment by jwb95** [comment]
Changing the optimizer to Adam and attenuating the learning rate fixed the problem for me.
